### PR TITLE
update mapping for shops in France (SIRENE) :

### PR DESCRIPTION
**Pull request by nuxper**
after analysis and test in Lyon :
* add 10.13B "Charcuterie" with a match to bucher or deli
* add 10.71B "Cuisson de produits de boulangerie" with a match to bakery or pastry
* add 47.11B "Commerce de détail en magasin non spécialisé à prédominance alimentaire" with a match to convenience/farm/supermarket/greengrocer/deli
* add matches to supermarket/farm/greengrocer/deli in 47.11C "Supérettes"
* add match to confectionery in 47.24Z "Commerce de détail de pain, pâtisserie et confiserie en magasin spécialisé"
* add matches to underwear/sewing in 47.51Z "Commerce de détail de textiles en magasin spécialisé"
* add 47.52A "Commerce de détail de quincaillerie, peintures et verres en petites surfaces (moins de 400 m²)"
* add 47.52B "Commerce de détail de quincaillerie, peintures et verres en grandes surfaces (400 m² et plus)"
* add matches to bed in 47.59A "Commerce de détail de meubles"
* add matches to bicycle in 47.64Z "Commerce de détail d'articles de sport en magasin spécialisé"
* add 47.74Z "Commerce de détail d'articles médicaux et orthopédiques en magasin spécialisé"
* add match second_hand=yes to 47.79Z "Commerce de détail de biens d'occasion en magasin"
* add match bar/pub/cafe to 56.10A "title": "Restauration traditionnelle
* add 68.31Z "Agences immobilières"
* add 79.12Z "Activités des voyagistes"
* add 96.01B "Blanchisserie-teinturerie de détail"
* add 96.04Z "Entretien corporel"

### CODE REVIEW

**Comment by frodrigo** [comment]
Do you pay attention to `trancheEffectifs` to limits?

File: merge_data/shop_FR.mapping.json [hunk 1]
```diff
@@ -1016,5 +1227,28 @@
...
+        "amenity": "swingerclub"
+      },
+      {
+        "shop": "massage"
+      }
+    ],
+    "generate":
+    {
+      "shop": "beauty;massage",
+      "fixme": "Select an amenity type. Use amenity=swingerclub for swinger clubs (club libertin)"
```
Comment (frodrigo) [inline merge_data/shop_FR.mapping.json:1251]: Do you thin it is generic to use 96.04Z for swingerclub?

**Comment by frodrigo** [comment]
Please, can you reformat your json (with jq).

File: merge_data/shop_FR.mapping.json (hunk 1 above)
Comment (nuxper) [reply merge_data/shop_FR.mapping.json:1251]: Yes I was quite surprised when testing 96.04Z in Lyon to find out that all swingerclubs in Lyon are categorised 96.04Z. I just checked two also in Bordeaux and Paris and it seems to be the same.

Comment (frodrigo) [reply merge_data/shop_FR.mapping.json:1256]: Ok, thank.

**Comment by frodrigo** [comment]
@nuxper please could you send an empty mail to osmose-contact@openstreetmap.fr I have complementary feedback on the SIREN integration you may be interested in.

**Comment by nuxper** [comment]
I've sent an email to the email address

**Comment by frodrigo** [comment]
Merged. Thank you.
Not yet deployed.

**Comment by frodrigo** [comment]
Deployed, update in progress.

cc @PanierAvide, note, more shop are coming

**Pull request merged**